SPECS/libtasn1/libtasn1.spec

```
@@ -24,7 +24,7 @@ The package contains libraries and header files for
 developing applications that use libtasn1.
 
 %prep
-%setup -q
+%autosetup -q
```

I think `%autosetup` doesn't have the `-q` option. Also, we still need `-p1` to actually apply the patch.;)

```suggestion
%autosetup -p1
```

Could you double-check this builds for both architectures?

---

Tracking here: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=257449&view=results